resources/js/common/utils/sortAchievements.ts

```diff
@@ -13,6 +13,9 @@ export function sortAchievements(
   sortOrder: AchievementSortOrder,
   eventAchievements?: App.Platform.Data.EventAchievement[],
 ): App.Platform.Data.Achievement[] {
+  // Create a copy to avoid mutating the original array.
+  const sortedAchievements = [...achievements];
```

Comment:
Nothing describes the problem being fixed, so I'm not sure how to test. If this is the only change, the list order may be affected, but not it's contents, so what was the problem?